File: Android Kotlin Retrofit Node Express- Orphanslife/app/src/main/java/com/kaligotla/orphanslife/model/Donation.kt
package com.kaligotla.orphanslife.model

import com.google.gson.annotations.SerializedName

data class Donation(
    @SerializedName("donation_id")
    val donation_id: Int,

    @SerializedName("amount")
    val amount: Double,

    @SerializedName("payment_status")
    val payment_status: String,

    @SerializedName("donation_created_at")
    val donation_created_at: String,

    @SerializedName("donation_updated_at")
    val donation_updated_at: String
)

File: MongoDB Express - API - Orphanslife/Data/model/Donation.js
const { Double } = require('mongodb');
const mongoose = require('mongoose')
const Schema = mongoose.Schema
const passportLocalMongoose = require('passport-local-mongoose');

var Donation = new Schema({
    donation_id:{
        type: Number
    },
    amount:{
        type: Number
    },
    payment_status:{
        type: String
    },
    sponsor_id:{
        type: Number
    },
    donation_created_at:{
        type: String
    },
    donation_updated_at:{
        type: String
    }
}, { collection: 'donation' })
  
Donation.plugin(passportLocalMongoose);
  
module.exports = mongoose.model('Donation', Donation)